README.adoc

```diff
@@ -8,15 +8,37 @@ https://coveralls.io/github/AY1920S2-CS2103T-W17-4/main?branch=master[image:http
 image:https://api.codacy.com/project/badge/Grade/a87445f769c04ad3863dad750abbb321["Codacy code quality", link="https://www.codacy.com/gh/AY1920S2-CS2103T-W17-4/main?utm_source=github.com&utm_medium=referral&utm_content=AY1920S2-CS2103T-W17-4/main&utm_campaign=Badge_Grade"]
 
 ifdef::env-github[]
-image::docs/images/Ui.png[width="600"]
+image::images/Ui.png[width="600"]
```

Comment:
Hmm, are you sure we should change this? might make the UI photo not show up again